### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by Daniil-Pankieiev**

### CODE REVIEW

**Comment by Daniil-Pankieiev** [comment]
image
image
image
image
image
image

**Review by roffi37 — APPROVED** [review]

File: taxi/urls.py [hunk 1]
```diff
@@ -16,8 +22,26 @@
...
+    path(
+        "manufacturer/delete/<int:pk>",
+        ManufacturerDeleteView.as_view(),
+        name="manufacturer-delete",
+    ),
     path("cars/", CarListView.as_view(), name="car-list"),
     path("cars/<int:pk>/", CarDetailView.as_view(), name="car-detail"),
+    path("cars/create", CarCreateView.as_view(), name="car-create"),
+    path("cars/<int:pk>/update", CarUpdateView.as_view(), name="car-update"),
+    path("cars/<int:pk>/delete", CarDeleteView.as_view(), name="car-delete"),
```
Comment (LiudmylaKulinchenko) [inline taxi/urls.py:44]: Missing a trailing `/` in the URL

**Review by LiudmylaKulinchenko — CHANGES_REQUESTED** [review]
Add Cancel button apart from Delete one. The Cancel button will lead to the previous page the user was on -- checklist

**Review by LiudmylaKulinchenko — APPROVED** [review]
Good job 🎈

**Pull request closed without merging**